elpy.el

```diff
@@ -537,6 +539,8 @@ virtualenv.
   :lighter " Elpy"
   (when (not (derived-mode-p 'python-mode))
     (error "Elpy only works with `python-mode'"))
+  (when (featurep 'xref)
+    (add-hook 'xref-backend-functions #'elpy--xref-backend nil t))
```

Comment:
I forgot to handle the keybindings...

`xref` commands should be bound by default (see [xref doc](https://www.gnu.org/software/emacs/manual/html_node/emacs/Looking-Up-Identifiers.html#Looking-Up-Identifiers)).
So we just need to define `M-.` and `M-,` to elpy functionalities if xref is not available.
The problem is that elpy do not use exactly the same shortcuts than xref (`elpy-goto-definition-other-window` is bound to `C-x 4 M-.`, but `xref-find-definition-other-window` is bound to `C-x 4 .`).
I added one additional shortcut to `xref-find-defintion-other-window` to avoid confusing people used to the elpy shortcuts.

I also add a paragraph for presenting the `xref` functionalities in the documentation.